clearbit/autocomplete.go

```diff
@@ -0,0 +1,39 @@
+package clearbit
+
+import (
+	"github.com/dghubble/sling"
+	"net/http"
+)
+
+const (
+	autoCompleteBase = "https://autocomplete.clearbit.com"
+)
+
+type AutocompleteItem struct {
+	Domain string `json:"domain"`
+	Logo   string `json:"logo"`
+	Name   string `json:"name"`
+}
+
+type AutocompleteSuggestParams struct {
+	Query string `url:"query"`
+}
+
```

Comment:
We should add comments to anything with uppercase. That way the Godocs will generate correctly:

```go
// AutocompleteService autocompletes a company name based on partial text.
type AutocompleteService struct {
```